src/components/Legend.js

```diff
@@ -22,9 +22,23 @@ export default class Legend {
       .attr('x2', '100%')
       .attr('y2', '0%');
 
-    legendGradient.append('stop')
-    .attr('offset', '0%')
-    .attr('style', `stop-color:${this.map.options.countries.attr.fill.min};stop-opacity:1`);
+    if (this.map.options.countries.attr.fill.minNegative &&
+        this.map.options.countries.attr.fill.maxNegative) {
+      legendGradient.append('stop')
+          .attr('offset', '0%')
+          .attr('style', `stop-color:${this.map.options.countries.attr.fill.maxNegative};stop-opacity:1`);
+
+      legendGradient.append('stop')
+          .attr('offset', '49%')
```

Comment:
oh - hrm - the problem in #27 will be aggravated by this change.  we'll need to update [``indiceChange()``](https://github.com/Packet-Clearing-House/maptable/blob/dev-28/src/components/Legend.js) to pull it all together so it works.